### DISCUSSION TITLE
Search for specific episodes in collapsed series

### DISCUSSION THREAD

**Discussion by kgraefe** [LeoKlaus/plappa#135] [Feature Requests]
When opening a collapsed series in the books tab or in the series tab it should have a search bar to help me find a specific episode.

Currently when searching for an episode in the books tab (with collapsed series) or the series tab it only finds the series. Tapping on it shows all episodes from that series. (Note: it even finds the series if the search term does not match the series title, so I guess it actually finds the episode)

A work-around is to temporary disable collapsed series and search in the books tab.

I'm on TestFlight version 1.3.0 (6).

**Comment by AgentP38** [comment]
Ping
